makefile

```diff
@@ -0,0 +1,21 @@
+# Builds everything in the universe.
+env-nfk:
+	$(info Copying .env.nfk to main .env)
+	cp .env .env.last;
+	cp .env.nfk .env;
+
+env-atb:
+	$(info Copying .env.atb to main .env)
+	cp .env .env.last;
+	cp .env.atb .env;
+
+clean-all:
+	$(info Cleaning app artifacts ...)
+	yarn clean;
+
+	$(info Deleting the build folder for ios and android...)
+	rm -rf ios/ios/build;
+	rm -rf android/app/build;
+
+#TODO:
+# Must create script to copy correct env files and GooglePlayServices files to right env directories
```

Comment:
Vi har allerede et script for å håndtere forskjellige environments som brukes i CI:
https://github.com/AtB-AS/mittatb-app/blob/master/scripts/override-environment.sh

Jeg tror vi er bedre tjent med å utvide funksjonalitet der. Vi baserer oss også på bash-scripts ellers, så vil helst ikke introdusere make i samme pipeline 😬 

Sånn sett vil jeg heller anbefale at NFK får egne mapper under `/env` som kan brukes til å override env/Firebase-plist.

F. eks. at disse config-filene ligger under `env/nfkDev`, og så kan dere override med `sh override-environment.sh nfkDev`

Det enkleste for Android er sikkert å sette opp egne build-variants i gradle/folder-struktur.